src/service/url-replacements-impl.js

```diff
@@ -313,6 +313,20 @@ export class GlobalVariableSource extends VariableSource {
       return Date.now();
     });
 
+    //Returns the human readable timestamp in format of 2011-01-01 11:11:11.
```

Comment:
The current code doesn't zero pad.

I'd highly prefer if we either

- don't do this
- or use `toISOString()` and be called `TIMESTAMP_ISO`

Formatting is highly arbitrary and people will want all kinds of format